functions/transformations/state_tracking.go

```diff
@@ -5,6 +5,8 @@ import (
 	"log"
 	"time"
 
+	"github.com/influxdata/flux/values"
```

Comment:
This import is out of order with the others.

Reply:
Fixed.